src/pygama/raw/orca/orca_SIS3316.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import sys
+from typing import Any
+
+import numpy as np
+
+from pygama.raw.orca.orca_header import OrcaHeader
+from pygama.raw.orca.orca_packet import OrcaPacket
+from pygama.raw.raw_buffer import RawBufferLibrary
+
+from .orca_base import OrcaDecoder, get_ccc
+
+log = logging.getLogger(__name__)
+
+
+class ORSIS3316WaveformDecoder(OrcaDecoder):
+    """Decoder for FlashCam ADC data written by ORCA."""
```

Comment:
Please update docstrings for all classes and methods with correct and complete documentation. Have a look at [our guide](https://legend-exp.github.io/pygama/releases/v1.0/developer.html#documentation) for how to write and build the documentation locally. You can also have a look at how other docstrings are formatted in pygama.